contract_price_revision/tests/test_contract_price_revision.py

```diff
@@ -82,3 +82,28 @@ def test_contract_price_revision_invoicing(self):
         self.assertEqual(len(lines), 2)
         lines = lines.filtered(lambda x: x.price_unit == 200.0)
         self.assertEqual(len(lines), 1)
+
+    def test_compute_variation_percent(self):
+        self.acct_line.price_unit = 200.0
+        self.acct_line.previous_price = 100.0
+        self.acct_line._compute_variation_percent()
```

Comment:
You shouldn't require to execute manually the compute method. It should be done automatically, or the `@api.depends` is not correct.

Reply:
I was getting a codecov error because the lines inside this @api.depends were not covered by the test:

line.variation_percent = (
    line.price_unit / line.previous_price - 1
) * 100

Should I delete it?

Reply:
This is not what I'm saying. Please re-read my comment.

Reply:
Ok, I think it's correct now.